grids/grid.css

```diff
@@ -0,0 +1,29 @@
+@import url("/style.css");
+
+.ct-grid {
+  display: grid;
+  /* width: 80vh; */
+  gap: 1rem;
+}
+.ct-grid-2 {
+  grid-template-columns: 1fr 1fr;
+}
+.ct-grid-3 {
+  grid-template-columns: 1fr 2fr 1fr;
```

Comment:
I think you should also make a variation for 3 equal-size columns `repeat(3,1fr)`  as that would be helpful to you.

Reply:
Okay I will make it tommo.